packages/components/src/popover/index.js

```diff
@@ -328,7 +329,7 @@ const Popover = (
 		}
 
 		return autoUpdate( usedRef, refs.floating.current, update );
-	}, [ anchorRef, anchorRect, getAnchorRect ] );
+	}, [ anchorRef, anchorRect, getAnchorRect, middlewares ] );
```

Comment:
This isn't a good idea, because it'll force `autoUpdate` on every render, but it seems we need a way to force `autoUpdate` to run if there's a change in middleware. Without it, the old middleware remains active.

Perhaps the middleware array should be memoized anyway.